dev-docs/workflows/bump-go-version.md

```diff
@@ -0,0 +1,16 @@
+# Bump Go version
+`govulncheck` from the bazel `check` target will fail if the Go version is outdated.
```

Comment:
That's not entirely correct @elchead. It will fail if our code is vulnerable, which is often the case when a patch version was released with security fixes.